src/components/GoogleSignIn/GoogleSignIn.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
Should the `GoogleSignIn` component be inside the `Login` component since that's the only place it will be needed?

Reply:
It will be in both the login and new user forms. I just haven't refactored the login form for it yet.

Reply:
Better idea - I'm going to remove it altogether from the profile flow. It will only live on the login page, and if they press it and don't have a profile, they will be redirected to the profile flow to fill out what's needed. We don't need two buttons.